lib/reporters/ApiaryReporter.js

```diff
@@ -40,7 +40,7 @@ function ApiaryReporter(emitter, stats, tests, config, runner) {
 
   this.configureEmitter(emitter);
 
-  logger.verbose(`Using '${this.type}' reporter.`);
+  logger.debug(`Using '${this.type}' reporter.`);
 
   if (!this.configuration.apiToken && !this.configuration.apiSuite) {
     logger.warn(`
```

Comment:
Should this message be a warning from the reporter? It is a reporter specific warning.